src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/PatchReservation.cs

```diff
@@ -49,6 +49,10 @@ public class PatchReservation : AzureReservationsCmdletBase
         [ValidateNotNull]
         public PSReservation Reservation { get; set; }
 
+        [Parameter(Mandatory = false)]
+        [ValidateNotNull]
+        public string Name { get; set; }
```

Comment:
I don't see updated help documentation for the `Name` parameter. Also, please provide an example in the help using this parameter.